docs_user/modules/con_openshift-preparation-for-block-storage-adoption.adoc

```diff
@@ -38,59 +34,37 @@ spec:
+====
 Transport protocols::
-Due to the specifics of the storage transport protocols some changes may be
-required on the {OpenShiftShort} side, and although this is something that must be
-documented by the Vendor here wer are going to provide some generic
-instructions that can serve as a guide for the different transport protocols.
+Due to the specifics of the storage transport protocols, some changes might be
+required for {OpenShiftShort}. Before making changes, review the following information:
 +
-Check the backend sections in your `cinder.conf` file that are listed in the
-`enabled_backends` configuration option to figure out the transport storage
-protocol used by the backend.
+* Check the back-end sections in your `cinder.conf` file that are listed in the
+`enabled_backends` configuration option to determine the transport storage
+protocol that the back end uses.
+* Depending on the back end, you can find the transport protocol by viewing the `volume_driver` or `target_protocol` configuration options.
+* The `icssid` service, `multipathd` service, and `NVMe-TCP` kernel modules start automatically on data plane nodes.
 +
-Depending on the backend, you can find the transport protocol:
-+
-* Looking at the `volume_driver` configuration option, as it may contain the
-protocol itself: RBD, iSCSI, FC...
-* Looking at the `target_protocol` configuration option
-+
-WARNING: Any time a `MachineConfig` is used to make changes to {OpenShiftShort}
-nodes the node will reboot!!  Act accordingly.
+[WARNING]
+If you use a `MachineConfig` to make changes to {OpenShiftShort} nodes, the nodes reboot.
 
 NFS::
-There is nothing to do for NFS. {OpenShiftShort} can connect to NFS backends without
-any additional changes.
+{OpenShiftShort} connects to NFS back ends without additional changes.
 
 RBD/Ceph::
-There is nothing to do for RBD/Ceph in terms of preparing the nodes, {OpenShiftShort}
-can connect to Ceph backends without any additional changes. Credentials and
-configuration files will need to be provided to the services though.
+{OpenShiftShort} connects to Ceph back ends without additional changes. You must provide credentials and configuration files to the services.
 
 iSCSI::
-Connecting to iSCSI volumes requires that the iSCSI initiator is running on the
-{OpenShiftShort} hosts where volume and backup services are going to run, because
-the Linux Open iSCSI initiator does not currently support network namespaces, so
-you must only run 1 instance of the service for the normal {OpenShiftShort} usage, plus
-the {OpenShiftShort} CSI plugins, plus the {OpenStackShort} services.
+To connect to iSCSI volumes, the iSCSI initiator must run on the
+{OpenShiftShort} hosts where volume and backup services are going to run. The Linux Open iSCSI initiator does not support network namespaces, so you must only run one instance of the service for the normal {OpenShiftShort} usage, as well as
+the {OpenShiftShort} CSI plugins and the {OpenShiftShort} services.
```

Comment:
@Akrog Is it correct to say "...as well as the OpenShift CSI plugins and the OpenShift services"?  Or should it say "OpenStack services"?

Reply:
It should be the OpenStack services.